addons/point_of_sale/static/src/xml/Screens/PaymentScreen/PaymentScreenElectronicPayment.xml

```diff
@@ -7,31 +7,31 @@
                 <div>
                     Payment request pending
                 </div>
-                <div class="button send_payment_request highlight" title="Send Payment Request" t-on-click="trigger('send-payment-request', props.line)">
+                <div class="button send_payment_request highlight" title="Send Payment Request" t-on-click="() => this.trigger('send-payment-request', props.line)">
```

Comment:
Shouldn't it be `this.props.line` instead of only `props.line`?

Reply:
It still works but it's kind of confusing. We use `this` to call `trigger` but to access a field, `this` is not necessary.

Reply:
`this` is necessary for trigger because otherwise trigger won't be bound to the component. You can absolutely use `this` everywhere in the future if you want (except when trying to access things that are defined in the rendering context like t-foreach loop variables or variables set with t-set)

Reply:
props is available in this context. But we agree it can be a bit confusing, perhaps adding this everywhere would be better to avoid confusion.